templates/web/base/admin/extra-metadata-form.html

```diff
@@ -6,7 +6,18 @@
         <div class="admin-hint"><p>[% loc('The ordering of this field on the report page. Fields are shown in ascending order according to this value.') %]</p></div>
         <label>
             [% loc('Order') %]
-            <input name="metadata[[% loop.index %]].order" data-field-name="order" type=text value="[% meta.order | html %]">
+            <input name="metadata[[% loop.index %]].order" data-field-name="order" type=text value="[% meta.order OR 0 | html %]">
+        </label>
+
+        <div class="admin-hint"><p>[% loc('Whether the field is displayed to the user, in which case it will be blank, otherwise it will be automatically populated. This field is set usually set automatically.') %]</p></div>
```

Comment:
Typo here: "set usually set". This doesn't explain the hidden field/server set difference, perhaps something like "Whether the field is displayed to the user, output in the form as a hidden field to be populated automatically, or only generated by the server upon Open311 submission" or similar?